test/test.js

```diff
@@ -0,0 +1,17 @@
+const test = require('tape');
+const supertest = require('supertest');
+const router = require('../src/controllers');
+
+test('test signup route ', (t) => {
+  supertest(router)
```

Comment:
are you sure that the test is run right???

I think you should make supertest for app.js file, not router.js 

```javascript
const router = require('../src/app');
supertest(router)
```